lib/src/cli/metrics/metrics_command.dart

```diff
@@ -58,10 +62,11 @@ Future<void> uploadMetrics(Options options) async {
   hierarchicalLoggingEnabled = true;
   log.level = options.verbose ? Level.INFO : Level.WARNING;
 
-  final skipUpload = isRealmCI || Platform.environment['CI'] != null || Platform.environment['REALM_DISABLE_ANALYTICS'] != null;
-  if (skipUpload && !isRealmCI) {
-    // skip early
-    log.info('Skipping metrics upload');
+  var skipUpload = (isRealmCI || Platform.environment['CI'] != null || Platform.environment['REALM_DISABLE_ANALYTICS'] != null) &&
+      Platform.environment['REALM_DEBUG_ANALYTICS'] == null;
```

Comment:
Could you check this logic? It looks strange. If it is not the CI then 'REALM_DISABLE_ANALYTICS' should be equal to null and not equal to null.

Reply:
the only change here is I introduced REALM_DEBUG_ANALYTICS which invokes the analytics logic but does not send it

Reply:
Oh, now I see that `REALM_DISABLE_ANALYTICS `<> `REALM_DEBUG_ANALYTICS`. The names are almost equal.